src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+body {
+  display: flex;
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.menu__head {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0 50px;
+  background-color: #fff;
+  height: 60px;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.menu__ul {
+  display: flex;
+  justify-content: space-between;
+  width: 641px;
```

Comment:
Do not hard code `width`

Reply:
done

Reply:
done, but shows correctly at: 1200 and 1024